src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -250,13 +250,13 @@ categories :
     - name        : MPIR_CVAR_CH4_OFI_MAX_VNIS
       category    : CH4_OFI
       type        : int
-      default     : 1
+      default     : 0
       class       : device
       verbosity   : MPI_T_VERBOSITY_USER_BASIC
       scope       : MPI_T_SCOPE_LOCAL
       description : >-
         If set to positive, this CVAR specifies the maximum number of CH4 VNIs
-        that OFI netmod exposes.
+        that OFI netmod exposes. The default is to be the same as MPIR_CVAR_CH4_NUM_VCIS.
```

Comment:
It would be good to add another line stating that if the user sets #VNIs > #VCIs, then we will internally set the VNIs to be the same as VCIs.

Reply:
done (to-be-pushed)